keyboards/45_ats/keymaps/default/keymap.c

```diff
@@ -0,0 +1,118 @@
+     * │    │ 1 │ 2 │ 3 │ 4 │ 5 │ 6 │ 7 │ 8 │ 9 │ 0 │      │   │End│
+     * ├────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴──┬───┘   ├───┤
+     * │      │ ( │ ) │ ; │ . │   │ - │ + │ * │ / │ =  │ ┌───┐ │   │
+     * ├────┬─┴─┬─┴──┬┴───┴───┴─┬─┴───┴──┬┴───┼───┴┬───┘ │   │ └───┘
+     * │    │   │    │          │        │    │    │ ┌───┼───┼───┐
+     * └────┴───┴────┴──────────┴────────┴────┴────┘ │   │   │   │
+     *                                               └───┴───┴───┘
+     */
+    [_FN] = LAYOUT_split_space(
+        _______,  KC_EXLM,  KC_AT,    KC_HASH,  KC_DLR,  KC_PERC,  KC_CIRC,  KC_AMPR,  KC_ASTR,  KC_LBRC,  KC_RBRC,  KC_BSLS,   _______,             KC_HOME,
+        _______,  KC_1,     KC_2,     KC_3,     KC_4,    KC_5,     KC_6,     KC_7,     KC_8,     KC_9,     KC_0,          KC_ENTER,                  KC_END,
+        _______,  KC_LPRN,  KC_RPRN,  KC_SCLN,  KC_DOT,  _______,  KC_MINS,  KC_PLUS,  KC_ASTR,  KC_SLSH,  KC_EQL,                     _______,      _______,
+        _______,  _______,  _______,                _______,            _______,            _______,       _______,          _______,  _______,   _______
+    ),
+
+    /* ALTERNATE Function layer (MOMENTARY)
+     * ┌────┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┐   ┌───┐
+     * │ PWR│F1 │F2 │F3 │F4 │F5 │F6 │   │   │   │   │PRV│NXT│   │VL+│
+     * ├────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴───┤   ├───┤
+     * │ SLP │F7 │F8 │F9 │F10│F11│F12│   │   │   │   │ PLAY │   │VL-│
+     * ├─────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴──┬───┘   ├───┤
+     * │ WAKE  │   │   │   │   │   │   │   │   │   │    │ ┌───┐ │   │
+     * ├────┬──┴─┬─┴──┬┴───┴───┴─┬─┴───┴──┬┴───┼───┴┬───┘ │   │ └───┘
+     * │RSET│    │    │          │        │    │    │ ┌───┼───┼───┐
+     * └────┴────┴────┴──────────┴────────┴────┴────┘ │   │   │   │
+     *                                                └───┴───┴───┘
+     */
+    [_LN] = LAYOUT_split_space(
+        KC_PWR,   KC_F1,    KC_F2,    KC_F3,    KC_F4,    KC_F5,    KC_F6,    _______,  _______,  _______,  _______,  KC_MPRV,  KC_MNXT,               KC_VOLU,
+        KC_SLEP,  KC_F7,    KC_F8,    KC_F9,    KC_F10,   KC_F11,   KC_F12,   _______,  _______,  _______,  _______,       KC_MPLY,                    KC_VOLD,
+        KC_WAKE,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,                    _______,       _______,
+        RESET,    _______,  _______,                 _______,            _______,            _______,       _______,          _______,  _______,   _______
+    )
+};
+
+//function for layer indicator LED
+uint32_t layer_state_set_user(uint32_t state)
+{
+    if (biton32(state) == 0) {
+    writePinHigh(D0);
+	} else {
+		writePinLow(D0);
+    }
+    if (biton32(state) == 1) {
+    writePinHigh(D1);
+	} else {
+		writePinLow(D1);
+    }
+    if (biton32(state) == 2) {
+    writePinHigh(D2);
+	} else {
+		writePinLow(D2);
+    }
+    return state;
+}
```

Comment:
```suggestion
layer_state_t layer_state_set_user(layer_state_t state) {
    writePin(D0, (get_highest_layer(state) == 0));
    writePin(D1, (get_highest_layer(state) == 1));
    writePin(D2, (get_highest_layer(state) == 2));
    return state;
}
```

Or, if you want them to light up regardless of the rest of the layers (eg each acts independant:

```suggestion
layer_state_t layer_state_set_user(layer_state_t state) {
    writePin(D0, layer_state_cmp(state, 0));
    writePin(D1, layer_state_cmp(state, 1));
    writePin(D2, layer_state_cmp(state, 2));
    return state;
}
```